README.md

```diff
@@ -6,20 +6,36 @@ on the [GitHub Applications Page](https://github.com/settings/applications).
 
 ## Basic Usage
 
-    use OmniAuth::Builder do
-      provider :github, ENV['GITHUB_KEY'], ENV['GITHUB_SECRET']
-    end
+Add to your `Gemfile`:
+gem 'omniauth-github'
```

Comment:
Add a blank line above and 4 space indent this to make MD format it as code?

    gem 'omniauth-github'